docs/sync.md

```
@@ -197,10 +196,10 @@ Security Notice:
   ╚════════════════════════════════════════════════════════╝
 ```
 
-### `mnemosyne sync serve`
+### `mnemosyne sync-serve`
 
 ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)